src/external/rawspeed/rawspeed-identify.cpp

```diff
@@ -119,7 +119,11 @@ int main(int argc, const char* argv[])
 
     fprintf(stderr, "Loading file: \"%s\"\n", argv[1]);
 
+    #if defined (_WIN32)
```

Comment:
I think it's safe to ignore Microsoft compilers. They are not capable of compiling darktable anyway.